specs/_features/eip7732/p2p-interface.md

```diff
@@ -0,0 +1,267 @@
+_ _[REJECT]_ The beacon block with root `data.beacon_block_root` passes validation. 
+- _[REJECT]_ The validator index is within the payload committee in `get_ptc(state, data.slot)`. For the current's slot head state. 
+- _[REJECT]_ The signature of `payload_attestation_message.signature` is valid with respect to the validator index.
+    
+###### `execution_payload_header`
+
+This topic is used to propagate signed bids as `SignedExecutionPayloadHeader`.
+
+The following validations MUST pass before forwarding the `signed_execution_payload_header` on the network, assuming the alias `header = signed_execution_payload_header.message`:
+
+- _[IGNORE]_ this is the first signed bid seen with a valid signature from the given builder for this slot.
+- _[IGNORE]_ this bid is the highest value bid seen for the pair of the corresponding slot and the given parent block hash.
+- _[REJECT]_ The signed builder bid, `header.builder_index` is a valid and non-slashed builder index in state.
+- _[IGNORE]_ The signed builder bid value, `header.value`, is less or equal than the builder's balance in state.  i.e. `MIN_BUILDER_BALANCE + header.value < state.builder_balances[header.builder_index]`.
+- _[IGNORE]_ `header.parent_block_hash` is the block hash of a known execution payload in fork choice.
+- _[IGNORE]_ `header.slot` is the current slot or the next slot. 
+- _[REJECT]_ The builder signature, `signed_execution_payload_header_envelope.signature`, is valid with respect to the `header_envelope.builder_index`.
+
+### The Req/Resp domain
+
+#### Messages
+
+##### BeaconBlocksByRange v3
+
+**Protocol ID:** `/eth2/beacon_chain/req/beacon_blocks_by_range/3/`
+
+[0]: # (eth2spec: skip)
+
+| `fork_version`           | Chunk SSZ type                |
+|--------------------------|-------------------------------|
+| `GENESIS_FORK_VERSION`   | `phase0.SignedBeaconBlock`    |
+| `ALTAIR_FORK_VERSION`    | `altair.SignedBeaconBlock`    |
+| `BELLATRIX_FORK_VERSION` | `bellatrix.SignedBeaconBlock` |
+| `CAPELLA_FORK_VERSION`   | `capella.SignedBeaconBlock`   |
+| `DENEB_FORK_VERSION`     | `deneb.SignedBeaconBlock`     |
+| `EIP7732_FORK_VERSION`   | `eip7732.SignedBeaconBlock`   |
+
+##### BeaconBlocksByRoot v3
+
+**Protocol ID:** `/eth2/beacon_chain/req/beacon_blocks_by_root/3/`
+
+Per `context = compute_fork_digest(fork_version, genesis_validators_root)`:
+
+[1]: # (eth2spec: skip)
+
+| `fork_version`           | Chunk SSZ type                |
+|--------------------------|-------------------------------|
+| `GENESIS_FORK_VERSION`   | `phase0.SignedBeaconBlock`    |
+| `ALTAIR_FORK_VERSION`    | `altair.SignedBeaconBlock`    |
+| `BELLATRIX_FORK_VERSION` | `bellatrix.SignedBeaconBlock` |
+| `CAPELLA_FORK_VERSION`   | `capella.SignedBeaconBlock`   |
+| `DENEB_FORK_VERSION`     | `deneb.SignedBeaconBlock`     |
+| `EIP7732_FORK_VERSION`   | `eip7732.SignedBeaconBlock`   | 
+
+
+##### BlobSidecarsByRoot v2
```

Comment:
These don't need a new version, the existing one is already versioned by `ForkDigest`-context. New version on the request is only needed if the request parameters or semantics change.